src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -69,4 +73,11 @@
 
   <Import Project="NativeDeps.csproj.include" />
 
+  <ItemGroup>
+    <Content Include="build\**\*.*">
```

Comment:
can you explain why exacty is this needed?

Reply:
Wouldn't it be better to list all the .targets files explicitly?
Like that it would be obvious that you're including .targets files and what platforms they'll be used for.

Btw, have you verified that the build/MonoAndroid/Grpc.Core.targets  file is actually needed?  AFAIK for the netstandard1.5 platform, the copying of the native libraries to the destination should work automagically without needing to provide the .targets file (as you can see we only have the .targets file for the net45 desktop platforms).

Reply:
Sure, explicitly listing the .targets files is fine, too.

Yes, the build/MonoAndroid/Grpc.Core.targets file is necessary, because in `Android App` projects builds use the `Build Action` called `AndroidNativeLibrary` to include native libraries in the build, see https://docs.microsoft.com/en-us/xamarin/android/platform/native-libraries.

There's another TODO here for `Android Class Library` projects which use the `Build Action` called `EmbeddedNativeLibrary`.